README.md

```diff
@@ -20,6 +20,9 @@ Shared configuration packages for frontend tools used by Pandell engineering tea
 - [`packages/typescript-config`](packages/typescript-config/)  
   Configuration for [typescript](https://www.typescriptlang.org/): _A strongly typed programming language that builds on JavaScript, giving you better tooling at any scale_
 
+- [`packages/webpack-config`](packages/webpack-config/)  
+  Configuration for [webpack](https://webpack.js.org/): _At its core, webpack is a static module bundler for modern JavaScript applications_
```

Comment:
⛏️ Nitpick: IMO we should include a text indicating this is a legacy package that will eventually be removed, and that is no longer recommended for new projects ([Vite](https://vite.dev/) should be used instead).

Reply:
I added a comment.